src/IECoreRenderMan/Object.cpp

```diff
@@ -40,6 +40,8 @@
 
 #include "RixPredefinedStrings.hpp"
 
+#include "fmt/format.h"
```

Comment:
This include doesn't appear to be necessary?

Reply:
Good catch - removed in https://github.com/GafferHQ/gaffer/pull/6416/commits/15805b218c349463710ec2fcf1d029651f1080e3.